tools/AutomationTestFramework/TestResources/RunbookTemplate.ps1

```diff
@@ -0,0 +1,10 @@
+Write-Output "JobId:$($PsPrivateMetaData.JobId.Guid)"
+$VerbosePreference = 'Continue'
+LoginWithConnection %LOGIN-PARAMS%
+
+%TEST-LIST%
+TestRunner $testList %LOGIN-PARAMS%
+
+Write-Verbose 'Resolve-AzureRmError Information'
+Write-Verbose '--------------------------------'
+Resolve-AzureRmError | Out-String | Write-Verbose
```

Comment:
Consider also adding another line with  ConvertTo-Json to provide additional object details for errors.